src/sentry/demo/data_population.py

```diff
@@ -1053,6 +1053,60 @@ def populate_connected_event_scenario_2(self, react_project: Project, python_pro
 
         self.log_info("populate_connected_event_scenario_2.finished")
 
+    def populate_connected_event_scenario_3(self, ios_project: Project):
+
+        ios_error = get_event_from_file("scen3/handled.json")
+        ios_transaction = get_event_from_file("scen3/ios_transaction.json")
+
+        self.log_info("populate_connected_event_scenario_3.start")
+
+        for (timestamp, day) in self.iter_timestamps(4):
+            transaction_user = self.generate_user()
+            trace_id = uuid4().hex
+            release = get_release_from_time(ios_project.organization_id, timestamp)
+            release_sha = release.version
+
+            old_span_id = ios_transaction["contexts"]["trace"]["span_id"]
+            root_span_id = uuid4().hex[:16]
+            duration = self.gen_frontend_duration(day)
+
+            trace = {
+                "trace_id": trace_id,
+                "span_id": root_span_id,
+            }
+
+            # React transaction
```

Comment:
@roggenkemper gotta update these comments haha

Reply:
I changed half of them but missed the other half